samtranslator/swagger/swagger.py

```diff
@@ -223,7 +224,7 @@ def _generate_integration_credentials(self, method_invoke_role=None, api_invoke_
 
     @staticmethod
     def _get_invoke_role(invoke_role):  # type: ignore[no-untyped-def]
-        CALLER_CREDENTIALS_ARN = "arn:aws:iam::*:user/*"
+        CALLER_CREDENTIALS_ARN = f"arn:{ArnGenerator.get_partition_name()}:iam::*:user/*"  # type: ignore[no-untyped-call]
```

Comment:
Isn't it easily typed? I'd assume `-> str`.

Reply:
```
samtranslator/translator/arn_generator.py:34: error: Unused "type: ignore" comment
samtranslator/translator/arn_generator.py:37: error: Function is missing a type annotation for one or more arguments
samtranslator/region_configuration.py:21: error: Unused "type: ignore" comment
samtranslator/region_configuration.py:55: error: "Session" has no attribute "get_partition_for_region"
samtranslator/sdk/parameter.py:79: error: Unused "type: ignore" comment
samtranslator/model/xray_utils.py:7: error: Unused "type: ignore" comment
samtranslator/model/apigatewayv2.py:225: error: Unused "type: ignore" comment
samtranslator/model/preferences/deployment_preference_collection.py:157: error: Unused "type: ignore" comment
samtranslator/model/apigateway.py:328: error: Unused "type: ignore" comment
samtranslator/model/eventsources/push.py:708: error: Unused "type: ignore" comment
samtranslator/model/eventsources/push.py:726: error: Unused "type: ignore" comment
samtranslator/model/eventsources/push.py:996: error: Unused "type: ignore" comment
samtranslator/model/api/api_generator.py:1146: error: Unused "type: ignore" comment
samtranslator/model/eventsources/cloudwatchlogs.py:41: error: Unused "type: ignore" comment
```